src/it/scala/com/wavesplatform/it/BalancesSpecification.scala

```diff
@@ -0,0 +1,318 @@
+package com.wavesplatform.it
+
+import java.io.IOException
+
+import com.wavesplatform.settings.Constants
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{BeforeAndAfterAll, FunSuite, Matchers, RecoverMethods}
+import scorex.transaction.TransactionParser.TransactionType
+
+import scala.concurrent.Await._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.Random
+
+class BalancesSpecification(allNodes: Seq[Node]) extends FunSuite with Matchers with ScalaFutures
+  with IntegrationPatience with BeforeAndAfterAll with RecoverMethods {
+  val sender = Random.shuffle(allNodes).head
+  val richBalance = sender.address
+
+  var first: String = _
```

Comment:
Offer me a replacement in this usage. Do I create a new address for each test with a new incoming balance transaction? I'm too lazy